### PR TITLE
Allow `@seek/indie-api-types` major through

### PR DESCRIPTION
**Pull request by 72636c**
This is tied to our AWS SDK upgrade: SEEK-Jobs/indie-api-types#911

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -41,7 +41,11 @@
       "semanticCommitType": "fix"
     },
     {
-      "excludePackageNames": ["@koa/cors", "@seek/ie-logging"],
+      "excludePackageNames": [
+        "@koa/cors",
+        "@seek/ie-logging",
+        "@seek/indie-api-types"
```
Comment (AaronMoat) [inline default.json:47]:
```suggestion
        "@seek/indie-api-types",
        "@seek/indie-mocks"
```

File: non-critical.json [hunk 2]
```diff
@@ -18,7 +18,11 @@
       "group": { "commitMessageTopic": "{{groupName}}" }
     },
     {
-      "excludePackageNames": ["@koa/cors", "@seek/ie-logging"],
+      "excludePackageNames": [
+        "@koa/cors",
+        "@seek/ie-logging",
+        "@seek/indie-api-types"
```
Comment (AaronMoat) [inline non-critical.json:24]:
```suggestion
        "@seek/indie-api-types",
        "@seek/indie-mocks"
```

File: third-party-major.json [hunk 3]
```diff
@@ -59,7 +59,11 @@
       "allowedVersions": "!/^4.21.0$/"
     },
     {
-      "excludePackageNames": ["@koa/cors", "@seek/ie-logging"],
+      "excludePackageNames": [
+        "@koa/cors",
+        "@seek/ie-logging",
+        "@seek/indie-api-types"
```
Comment (AaronMoat) [inline third-party-major.json:65]:
```suggestion
        "@seek/indie-api-types",
        "@seek/indie-mocks"
```

**Review by AaronMoat — APPROVED** [review]
Another one incoming...

File: default.json (hunk 1 above)
Comment (72636c) [reply default.json:47]: 👍 I omitted because it looked like there was one consumer, but doesn't hurt

**Pull request merged**